.gitignore

```diff
@@ -21,3 +21,6 @@ current
 npm-shrinkwrap.json
 package-lock.json
 yarn.lock
+
+#ide and editor configurations
+.idea/*
```

Comment:
please revert this; IDE-specific ignores should go in your personal global gitignore, rather than polluting every repo you touch :-)
```suggestion
```